src/components/ButtonPanel.js

```diff
@@ -18,40 +18,41 @@ function ButtonPanel(props) {
   return (
     <div style={panelStyles} className="calc-panel">
       <div style={rowStyles}>
-        <Button color="grey" name="AC" />
-        <Button color="grey" name="+/-" />
-        <Button color="grey" name="%" />
-        <Button name="÷" />
+        <Button color="grey" name="AC" onClick={clickHandler} />
+        <Button color="grey" name="+/-" onClick={clickHandler} />
+        <Button color="grey" name="%" onClick={clickHandler} />
+        <Button name="÷" onClick={clickHandler} />
       </div>
       <div style={rowStyles}>
-        <Button color="grey" name="7" />
-        <Button color="grey" name="8" />
-        <Button color="grey" name="9" />
-        <Button name="X" />
+        <Button color="grey" name="7" onClick={clickHandler} />
+        <Button color="grey" name="8" onClick={clickHandler} />
+        <Button color="grey" name="9" onClick={clickHandler} />
+        <Button name="X" onClick={clickHandler} />
       </div>
       <div style={rowStyles}>
-        <Button color="grey" name="4" />
-        <Button color="grey" name="5" />
-        <Button color="grey" name="6" />
-        <Button name="-" />
+        <Button color="grey" name="4" onClick={clickHandler} />
+        <Button color="grey" name="5" onClick={clickHandler} />
+        <Button color="grey" name="6" onClick={clickHandler} />
+        <Button name="-" onClick={clickHandler} />
       </div>
       <div style={rowStyles}>
-        <Button color="grey" name="1" />
-        <Button color="grey" name="2" />
-        <Button color="grey" name="3" />
-        <Button name="+" />
+        <Button color="grey" name="1" onClick={clickHandler} />
+        <Button color="grey" name="2" onClick={clickHandler} />
+        <Button color="grey" name="3" onClick={clickHandler} />
+        <Button name="+" onClick={clickHandler} />
       </div>
       <div style={rowStyles}>
-        <Button wide color="grey" name="0" />
-        <Button color="grey" name="." />
-        <Button name="=" />
+        <Button wide color="grey" name="0" onClick={clickHandler} />
+        <Button color="grey" name="." onClick={clickHandler} />
+        <Button name="=" onClick={clickHandler} />
       </div>
```

Comment:
If we could map through an array and build this 🤔 that would avoid so much duplicate code.

Reply:
I'll try something about that. Great sugestion!